components/Elevation/src/UIColor+MaterialElevation.h

```diff
@@ -30,6 +30,23 @@
  */
 - (nonnull UIColor *)mdc_resolvedColorWithElevation:(CGFloat)elevation;
 
+/**
+ Returns a color that takes the specified elevation value and traits into account when there is a
+ color appearance difference between current traits and previous traits. When userInterfaceStyle is
+ UIUserInterfaceStyleDark in currentTraitCollection, elevation will be used to resolve the color.
+ Negative elevation is treated as 0.
```

Comment:
Should we separate these on new lines to make it a little more readable?

Reply:
Addressed in 75d9d11dfb3fd470687a570978baba739a411c59.